packages/tags/README.md

```diff
@@ -0,0 +1,136 @@
+## Description
+
+`sp-tags` and `sp-tag` elements allow users to categorize content. They can represent keywords or people, and are grouped to describe an item or a search request.
+
+## To do:
+
+Keyboard interactions
```

Comment:
I like this To do pattern!

Reply:
Oh this was supposed to be taken out...whoops.

Reply:
Removed.